x-pack/plugins/security_solution/public/common/components/accessibility/tooltip_with_keyboard_shortcut/index.tsx

```diff
@@ -10,7 +10,7 @@ import React from 'react';
 
 import * as i18n from './translations';
 
-interface Props {
+export interface TooltipWithKeyboardShortcutProps {
```

Comment:
move it too packages tgrid

Reply:
since we added the feature flag, we can keep the copy since we are going to delete when we are only using EuiDatagrid